### PR TITLE
home: comparison metrics are underneath metric descriptions

### PR DESCRIPTION
**Pull request by maxwofford**
screenshot 2015-08-18 at 4 10 55 pm

### CODE REVIEW

**Comment by MaxWofford** [comment]
Closes #132

**Comment by MaxWofford** [comment]
@zachlatta Would you please review this?

**Comment by zachlatta** [comment]
For the minorities number, can you use the number of underrepresented
minorities in industry? That seems to be what most people use.

Also, it looks a bit unbalanced having that extra text under two of the
headings, but not the third (the text is also a little small). Any thoughts
on how we can better present this information?

On Tue, Aug 18, 2015 at 5:01 PM, Max Wofford <notifications@github.com>
wrote:

> @zachlatta <https://github.com/zachlatta> Would you please review this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/website/pull/142#issuecomment-132395154>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by MaxWofford** [comment]
How's this?
screenshot 2015-08-22 at 6 00 59 pm

**Comment by zachlatta** [comment]
Looks good. Can you make sure to update that minorities number as well?

And perhaps include citations?

On Sat, Aug 22, 2015 at 6:02 PM, Max Wofford <notifications@github.com>
wrote:

> How's this?
> [image: screenshot 2015-08-22 at 6 00 59 pm]
> <https://cloud.githubusercontent.com/assets/5891442/9426461/d84108de-48f7-11e5-91e9-70065f97f03a.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/website/pull/142#issuecomment-133768375>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by zachlatta** [comment]
@jonleung can you grab the citation for the minorities number?

On Sat, Aug 22, 2015 at 6:25 PM, Zach Latta <zach@hackedu.us> wrote:

> Looks good. Can you make sure to update that minorities number as well?
>
> And perhaps include citations?
>
> On Sat, Aug 22, 2015 at 6:02 PM, Max Wofford <notifications@github.com>
> wrote:
>
>> How's this?
>> [image: screenshot 2015-08-22 at 6 00 59 pm]
>> <https://cloud.githubusercontent.com/assets/5891442/9426461/d84108de-48f7-11e5-91e9-70065f97f03a.png>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hackedu/website/pull/142#issuecomment-133768375>.
>>
>
>
>
> --
> Zach Latta
> hackEDU | zach@hackedu.us
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
Here's what I found about the minority statistics from Googling:

> The percentage of African-American test-takers also held steady at around 4 percent while the share of Hispanic participants increased slightly to about 9 percent from 8 percent. In comparison, the Western Interstate Commission for Higher Education calculates that, nationwide, black and Hispanic students made up 14 percent and 19 percent, respectively, of the high school class of 2014.

I got this from http://www.edweek.org/ew/articles/2014/12/19/more-students-but-few-girls-minorities-took-ap-computer.html

**Comment by MaxWofford** [comment]
Based on http://research.collegeboard.org/programs/ap/data/participation/ap-2014, 19.98% of AP CS test takers were female.

**Comment by zachlatta** [comment]
@jonleung I'm referring to the other minorities number, mentioned in https://github.com/hackedu/website/pull/142#issuecomment-133080299.

**Comment by MaxWofford** [comment]
@zachlatta Woah, how did you get the comment number of that?!

**Comment by zachlatta** [comment]
Like this/;

recorded

**Comment by zachlatta** [comment]
@MaxWofford can you change the minorities number to the percentage in industry, per https://github.com/hackedu/website/pull/142#issuecomment-134854964?

@jonleung can you get @MaxWofford the source for the number you gave me a while back.

**Comment by MaxWofford** [comment]
@zachlatta On it

**Comment by jonleung** [comment]
@zachlatta I think I would be more comfortable using the [Education Week](http://www.edweek.org/ew/articles/2014/12/19/more-students-but-few-girls-minorities-took-ap-computer.html) statistic because it cites the number of AP CS test takers.

We are using an analogous statistic for the female statistic.

The number is from [USA TODAY](http://www.usatoday.com/story/tech/2014/12/29/20868353/) and talks about the percentage of minorities at leading technology companies. Which is very far removed from high schools and only accounts for _**leading**_ tech companies:

> SAN FRANCISCO — In 2014, leading technology companies released data showing they vastly underemploy African Americans and Hispanics. Those groups make up 5% of the companies workforce, compared to 14% nationally.

**TLDR**: I think we should use the Education Week numbers.

**Comment by zachlatta** [comment]
Okay, I'm alright with that.

**Comment by MaxWofford** [comment]
If that's the case, the metric for AP CS test takers is already in there.
